pkg/services/alerting/notifiers/slack.go

```diff
@@ -174,9 +225,32 @@ func (sn *SlackNotifier) Notify(evalContext *alerting.EvalContext) error {
 		})
 	}
 
-	message := sn.Mention
+	message := ""
```

Comment:
Should probably be a string builder? Also maybe maybe something like:

```
	foo := ""
	msgAddSpace := func() {
		if foo != "" {
			foo += " "
		}
		return
	}
	msgAddSpace()
``` 

to reduce repetition,  bit with stringbuilder.

Reply:
Thanks @kylebrandt - will look into it.